.remarkignore

```diff
@@ -0,0 +1,19 @@
+# Changelog is autogenerated
+CHANGELOG.md
+example-changelog.md
+vendor
+dist
+build
+.github
+
+# Temporary disabled packages
+# we will fix them incrementally
```

Comment:
Well, I think I will do it as a side job. I want to focus on some kind of programming between all those meetings, discussions, and code reviews, so for my brain this a some kind of the rest.

My first intention was to do this all-in-one piece, but then I realized that it would be a much more bigger PR and too many files to review. And I know you will be the first to make complaints about it :-)

So thank you for your suggestion even though this is not in our current scope ❤️